databricks_cli/secrets/cli.py

```diff
@@ -316,7 +336,7 @@ def get_acl(api_client, scope, principal, output):
 @debug_option
 @profile_option
 @eat_exceptions
-def secrets_group():
+def secrets_group():  # pragma: no cover
```

Comment:
is it intended?

Reply:
I think this was a result of resolving the merge conflict? Not my change, but certainly came from master